content/docs/Reference/env-variables.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Environment Variables"
+linkTitle: "Environment Variables"
+date: 2019-03-30
+weight: 4
+description: >
+  Reference for common environment variables
```

Comment:
I recommend making this more task oriented, as follows:

Configure common environment variables